src/api/getChangelogData.js

```diff
@@ -42,11 +43,29 @@ function getVersionChangelog(
         getVersionChangelogFileNames(config, stringifyVersion(version))
           .map((entryFileName) => readFileSync(entryFileName).toString())
           .map(jsYaml.safeLoad)
+          .sort(sortByComponent.bind(null, config))
       ), 'kind')
     }
   };
 }
 
+function sortByComponent(config: ConfigType, entry1: EntryType, entry2: EntryType) {
```

Comment:
Hmm, so this mixes the concerns of: 1) sorting and 2) getting the component from the config. I wonder if some kind of `getComponent(config, entry)` method would work / be more intuitive?